commands/server/start.go

```diff
@@ -48,7 +60,7 @@ func StartCmd(gen AppGenerator, logger log.Logger, home string, args []string) e
 
 	svr, err := server.NewServer(addr, "socket", app)
 	if err != nil {
-		return errors.Errorf("Error creating listener: %v\n", err)
+		return fmt.Errorf("Error creating listener: %v\n", err)
```

Comment:
If you fix this, let's use our `weave/errors` package, like `errors.Wrap`

Reply:
There's no real reason as we are failing on start, so none of this matters? But sure, let's be consistent